run/README.md

```diff
@@ -0,0 +1,86 @@
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
+
+# Google Cloud Run Ruby Samples
+
+[Cloud Run][run_docs] runs stateless [containers](https://cloud.google.com/containers/) on a fully managed environment or in your own GKE cluster.
+
+## Samples
+
+|                 Sample                  |        Description       |     Deploy    |
+| --------------------------------------- | ------------------------ | ------------- |
+|[Hello World][helloworld]  | Quickstart | [<img src="https://storage.googleapis.com/cloudrun/button.svg" alt="Run on Google Cloud" height="30"/>][run_button_helloworld] |
+
+For more Cloud Run samples beyond Ruby, see the main list in the [Cloud Run Samples repository](https://github.com/GoogleCloudPlatform/cloud-run-samples).
+
+## Setup
```

Comment:
Any prereqs? Install ruby? Which version?

Reply:
Info provided in parent readme